Kernel/CommandLine.cpp

```diff
@@ -4,13 +4,15 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <AK/StringBuilder.h>
 #include <Kernel/CommandLine.h>
 #include <Kernel/Panic.h>
 #include <Kernel/StdLib.h>
 
 namespace Kernel {
 
 static char s_cmd_line[1024];
+static const char* s_embedded_cmd_line = "";
```

Comment:
Can we make this

```suggestion
static constexpr StringView s_embedded_cmd_line = "";
```

? Then we shouldn't need the StringView construction in `build_commandline()`.